account/accounts.go

```diff
@@ -245,6 +248,27 @@ func (m *Manager) SelectAccount(walletAddress, chatAddress, password string) err
 	return nil
 }
 
+// SetChatAccount initializes selectedChatAccount with privKey
+func (m *Manager) SetChatAccount(privKey *ecdsa.PrivateKey) error {
```

Comment:
Returning `error` seems obsolete because it is never set.

Reply:
@adambabik you are right, it was used before but with the new version it's not even used, I'm removing it